backend/app/models/visualizations.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+from typing import Optional, List
+
+from beanie import Document, View, PydanticObjectId
+from pydantic import BaseModel, Field
+
+from app.models.files import FileContentType
+from app.models.users import UserOut
+
+
+class VisualizationsBase(BaseModel):
+    name: str = "N/A"
+    description: Optional[str] = None
+
+
+class VisualizationsIn(VisualizationsBase):
+    pass
+
+
+class VisualizationsPatch(BaseModel):
+    name: Optional[str]
+    description: Optional[str]
+
+
+class VisualizationsDB(View, VisualizationsBase):
+    creator: UserOut
+    created: datetime = Field(default_factory=datetime.utcnow)
+    modified: datetime = Field(default_factory=datetime.utcnow)
+    bytes: int = 0
+    content_type: FileContentType = FileContentType()
+
+    class Settings:
+        name = "Visualizations"
```

Comment:
maybe lowercase for the collection name just follow other conventions?

Reply:
done